### PR TITLE
feat: [CCM-7413]: Added sustainability widgets across CCM 

### PR DESCRIPTION
**Pull request by rishik30**
##### Screenshots:
Screenshot 2022-04-27 at 5 32 02 PM
Screenshot 2022-04-27 at 5 32 29 PM
Screenshot 2022-04-27 at 5 32 55 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by rishik30** [comment]
trigger cypress

**Comment by rishik30** [comment]
trigger cypress

**Comment by rishik30** [comment]
retrigger jest

**Comment by rishik30** [comment]
retrigger jest

**Comment by rishik30** [comment]
retrigger jest

**Comment by rishik30** [comment]
retrigger jest

**Comment by rishik30** [comment]
retrigger jest

File: src/modules/75-ce/common/images/gray-leaf.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<svg width="597" height="597" viewBox="0 0 597 597" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (rajarshee-harness) [inline src/modules/75-ce/common/images/gray-leaf.svg:1]: Run SVGO?

**Review by rajarshee-harness — APPROVED** [review]

**Comment by rishik30** [comment]
retrigger jest

**Review by jyotiarora-harness — APPROVED** [review]

**Pull request merged**